src/renderer/pages/Settings/Networks/model/networks-overview-model.ts

```diff
@@ -1,8 +1,45 @@
-import { sample } from 'effector';
+import { sample, combine } from 'effector';
 
-import { networksFilterModel, activeNetworksModel, inactiveNetworksModel } from '@features/network';
+import { ChainId } from '@shared/core';
+import type { ConnectionItem } from '@features/network/NetworkSelector';
+import {
+  networksFilterModel,
+  activeNetworksModel,
+  inactiveNetworksModel,
+  networkSelectorUtils,
+} from '@features/network';
+import { Predicates } from '../lib/constants';
 
 sample({
```

Comment:
Oh by the way, now we have computed stores, let's move `sample` down below as we do in other model
`events > stores > effects > combines > samples`